src/screens/Donate/Donate.style.tsx

```diff
@@ -113,3 +128,23 @@ export const SectionHeader = styled.h2`
   font-weight: bold;
   line-height: 1.4;
 `;
+
+export const StyledIframe = styled.iframe`
+  height: 775px;
+  width: 100%;
+  max-width: 560px;
+  border: 1px solid ${COLOR_MAP.GRAY.LIGHT};
+  border-radius: 8px;
+
+  @media (min-width: ${materialSMBreakpoint}) {
+    height: 725px;
+  }
+
+  @media (min-width: ${mobileBreakpoint}) {
+    height: 750px;
+  }
+
+  @media (min-width: 1200px) {
+    height: 700px;
+  }
+`;
```

Comment:
The height is a bit tricky cos the iframe adjusts its height dynamically. So I tried to get it to adjust with the iframe as much as possible.